pkg/interface/src/views/components/ImageInput.tsx

```diff
@@ -43,10 +40,55 @@ export function ImageInput(props: ImageInputProps): ReactElement {
     }
   }, [ref.current, uploadDefault, canUpload, setValue]);
 
-  const onClick = useCallback(() => {
+  const clickUploadButton = useCallback(() => {
     ref.current?.click();
   }, [ref]);
 
+  const Prompt = () => (
+    <Text
+      black
+      fontWeight='500'
+      position='absolute'
+      left={2}
+      top={2}
+      style={{ pointerEvents: 'none' }}
+    >
+      Paste a link here, or{' '}
+      <Text
+        fontWeight='500'
+        cursor='pointer'
+        color='blue'
+        style={{ pointerEvents: 'all' }}
```

Comment:
just curious, wat mean?

Reply:
This lets the user click on the link itself instead of focusing the field beneath it. A bit of trickery, can see if it will still work without.

Reply:
oh, cool